### ISSUE TITLE
[REF] convertIfcSectionedSolidHorizontal 

### ISSUE THREAD

**Issue by SamuilsRulovs** [tumcms/Open-Infra-Platform#439]
**File which is being refactored**
core/IfcGeometryConverter/SolidConverter
**Functions**
convertIfcSectionedSolidHorizontal

**Comment by jschlenger** [comment]
I guess this issue can be closed, right?

**Comment by pjanck** [comment]
No, not yet. This changed quite a lot with IFC4x3_RC2, so it would need at least a check if not a rework.
